proto/hummock.proto

```diff
@@ -54,6 +60,19 @@ message HummockVersion {
   uint64 safe_epoch = 5;
 }
 
+message HummockVersionDelta {
+  message LevelDeltas {
+    repeated LevelDelta level_deltas = 1;
+  }
+  uint64 id = 1;
+  // Levels of each compaction group
```

Comment:
Should we also store the `prev_id`?

Reply:
> Should we also store the `prev_id`?

seems that we do need to store `prev_id`. btw, is `prev_id` usually the same as `id - 1`?

Reply:
Currently it is `id -1`. But I don't think we should make this assumption because we only need version id to be monotonically increasing. Continuousness is not a strict requirement.

Reply:
> Currently it is `id -1`. But I don't think we should make this assumption because we only need version id to be monotonically increasing. Continuousness is not a strict requirement.

fixed